Documentation/Getting-Started/ceph-openshift.md

```diff
@@ -21,6 +21,9 @@ oc create -f operator-openshift.yaml
 oc create -f cluster.yaml
 ```
 
+## Helm Installation
+
+Configuration required for Openshift is automatically created by the Helm charts, such as the SecurityContextConstraints. See the [Rook Helm Charts](../Helm-Charts/helm-charts.md].
```

Comment:
The closing parenthesis is incorrect, and need a newline before the next section.
```suggestion
Configuration required for Openshift is automatically created by the Helm charts, such as the SecurityContextConstraints. See the [Rook Helm Charts](../Helm-Charts/helm-charts.md).

```